tests/TestPassphraseGenerator.cpp

```diff
@@ -52,3 +53,13 @@ void TestPassphraseGenerator::testWordCase()
     QRegularExpression regex("^(?:[A-Z][a-z-]* )*[A-Z][a-z-]*$");
     QVERIFY2(regex.match(passphrase).hasMatch(), qPrintable(passphrase));
 }
+void TestPassphraseGenerator::testUniqueEntriesInWordlist()
```

Comment:
Missing empty line here. Probably would be fixed by running code format.

Reply:
done